src/test/resources/copyGraph.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+wget https://github.com/amcp/dynamodb-import-export-tool/archive/createTable.zip
```

Comment:
Use signed and published artifacts in maven to reduce build time and continue the chain of trust.